TypeInjections/Tool/Tool.fsproj

```diff
@@ -29,6 +29,7 @@
         <PackageReference Include="CommandLineParser.FSharp" Version="2.9.2-ci-210"/>
         <PackageReference Include="DafnyPipeline" Version="3.6.0.40511"/>
         <PackageReference Include="LibGit2Sharp" Version="0.27.0-preview-0182"/>
+        <PackageReference Include="CommandLineParser.FSharp" Version="2.9.2-ci-210" />
```

Comment:
This dependency is already included on line 29

Reply:
Fixed